crates/devolutions-pedm-shared/src/policy.rs

```diff
@@ -360,7 +356,7 @@ impl Identifiable for Profile {
 impl Default for Profile {
     fn default() -> Self {
         Self {
-            id: Id("default".to_owned()),
+            id: Id(Uuid::new_v4()),
```

Comment:
Is it caused by this implementation of the Default trait? Does that mean we’ll change the default value in the OpenAPI document at each generation? I’m thinking that the OpenAPI document should not include a default value.

Reply:
Yes, you're right, and I hadn't considered why this was here

We're forced to have a `Default` here; I'll see if I can remove it since it's really not needed in the documentation. This API is expressly private.

Otherwise, I'll change it to an empty GUID.

Reply:
I've changed it to `uuid::default()` so we get an empty GUID in the doc examples.

It's OpenAPI that forces a `Default` implementation here; perhaps we can disable this but for now I think this is good.

Next time I'll separate commits for the actual changes versus API generation, sorry about that